mos_tests/neutron/python_tests/test_dvr.py

```diff
@@ -64,3 +65,180 @@ def test_vm_connectivity_wo_floating(self, variables):
             security_groups=[self.security_group.id])
 
         self.check_ping_from_vm(server, vm_keypair=self.instance_keypair)
+
+    def try_to_update_router_to_centralized(self, router_id):
```

Comment:
Do you expect exception? If yes - let's use this check - https://pytest.org/latest/assert.html#assertions-about-expected-exceptions